### ISSUE TITLE
Label alignment off

### ISSUE THREAD

**Issue by rhssk** [vivien/i3blocks#214]
When using Font Awesome, any icon from fa-volume family (except fa-volume-control-phone) breaks text alignment as seen in here: https://i.imgur.com/eu1aC4y.png.

I'm using i3blocks-gaps-git, but for testing purposes I swapped that out with [i3blocks 1.4-3](https://aur.archlinux.org/packages/i3blocks/) and the issue remained. System has Font Awesome 4.7.0 installed. I downgraded to 4.6.3 which is a version I know for sure worked previously and the alignment was still off.

The weird thing is that only 3 icons in total are not working - fa-volume-up, fa-volume-down and fa-volume-off. Now I know that this sounds more like an issue with Font Awesome, but as I said, 4.6.3 still doesn't work (which did before). And their issue tracker doesn't have anything relevant. i3blocks AUR packages were last updated 4-5 months ago so it shouldn't be something a new update brought in.

Suggestions?

**Comment by kb100** [comment]
If there are multiple fonts on your system that offer volume control characters, i3 may pick the wrong one or do something nonsensical if you don't tell it what font to use.

Try using `markup=pango` and surrounding the volume icons like `<span font='FontAwesome'></span>`

**Comment by rhssk** [comment]
Thanks for the tip, I tried it but got a block with nothing but whitespace. Based on your answer I checked my current font in FontForge and it indeed already has characters for `U+F026` etc. First search result for "pango span Font Awesome" suggested putting Font Awesome after i3config bar section's font.
So instead of
`font pango:Terminus 12`
I know have
`font pango:Terminus, Font Awesome 12`
which fixes my problem. Thank you for helping, issue can be closed.
